### DISCUSSION TITLE
DOM Element Error

### DISCUSSION THREAD

**Discussion by Dreitser** [jeffreylanters/react-unity-webgl#265] [Help Wanted!] [answered]
I'm running vanilla create-react-app and Unity 2019.4.29f1 LTS and I'm getting this error. I can X it out and seems to work OK but not desired. 
<img width="449" alt="Screen Shot 2021-11-19 at 12 44 16 AM" src="https://user-images.githubusercontent.com/2120817/142593018-bf67cdaa-98cd-4de6-ab18-037c18063157.png">

<img width="1019" alt="Screen Shot 2021-11-19 at 12 44 28 AM" src="https://user-images.githubusercontent.com/2120817/142593005-c2d2fa26-f040-48ae-849d-1e5296baedaa.png">

**Comment by jeffreylanters** [comment, marked as answer]
Hi Lex! This issue is not related to the module, but I can help you out. The second parameter of ReactDOM's render method should be a reference to an HTML Element. I can't see the rest of your render method's implementation, so I'm not completely sure what you've entered. But an example could look like this;

```ts
// Get a reference to an HTML Element using its ID attribute in your HTML.
const rootElement = document.getElementById("root");

// Then render the app onto the document using the render method.
ReactDOM.render(
  <StrictMode>
    <App />
  </StrictMode>,
  rootElement
);
```

Let me know if this helps.

**Reply by Dreitser** [reply]
I few gotchas I found that maybe may help others get started.
1. There is separate documentation for Version 7.x https://github.com/jeffreylanters/react-unity-webgl/tree/7.1.13
2. UnityContent (or UnityContext in version 8.x) points to files generated by Unity WebGL Build
